docs/source/architecture/servicefees.rst

```diff
@@ -0,0 +1,79 @@
+
+**Minimum fee rate**
+
+XX GWEI (~$X.XX x10^-X) *per period*
+
+XX,XXX GWEI ($X.XXX) *per year*
+
+**Maximum fee rate**
+
+X,XXX GWEI (~$X.XX x10^-X) *per period*
+
+XXX,XXX GWEI ($X.XXX) *per year*
+
+**Default fee rate**
+
+XXX GWEI (~$X.XX x10^-X) *per period*
+
+XX,XXX GWEI ($X.XXXX) *per year*
+
+*1 GWEI = 10^-9 ETH*
+
+*USD conversion utilizes the ETHUSD 100-day rolling average of 1 ETH = $222 (08.08.20)*
+
+Note that the minimum and maximum fee rate are an upper and lower bound to constrain the fee rate a staker may offer. The default fee rate is the rate that will be displayed and offered to Alices if the staker chooses not to configure this parameter themselves.
+
+The fee range must be adhered to in identical fashion by all NuCypher stakers, regardless of their stake size or capacity. The fee range applies to all sharing policies, irrespective of the volume of re-encryption requests or other distinguishing attributes. It also applies equally to all periods in the future, until the moment that the global fee range’s parameters are adjusted or the range is removed, via official governance processes (see below). If an update of this sort occurs, sharing policies that were previously established, but have not yet expired, should not have the per-period fee rate retroactively modified.
+
+Governance & pricing paper
+---------------------------------------
+
+In order to successfully execute the ``PolicyManager.sol`` contract, the global fee range must be adhered to by the Ursula (and Alice). Failing this, the contract will throw up an error and it will not be possible to commence a commercial engagement or pay/receive fees. Attempts to circumvent NuCypher’s smart contracts are likely to be futile (except perhaps with limited, existing, trust-heavy relationships), given the requirement of coordinated modification and redeployment by network users and a critical mass of other stakers.
+
+Given its high enforceability, the presence of an inflexible fee range dictating the bounds of every transaction is arguably the most critical component of the NuCypher protocol’s economic design and parametrization, particularly over the long-term and with respect to the sustainability of the network. From a governance perspective, it is also amongst the most malleable. If a quorum of stakers wish to set prices outside the range, then they have the right to lobby and propose a widening of the global fee range, its removal altogether, or some other design modification (e.g. narrowing the range). They may do so via the NuCypher DAO – the owner of all NuCypher smart contracts, and controlled by stakers in proportion to their stake size. See <LINK> for guidance on the NuCypher DAO and official NuCypher governance processes.
```

Comment:
all NuCypher smart contracts, and controlled by stakers in proportion to their stake size. -> all NuCypher smart contracts -  by submitting a proposal which is voted on by stakers weighted in proportion to their stake size.
 
```suggestion
Given its high enforceability, the presence of an inflexible fee range dictating the bounds of every transaction is arguably the most critical component of the NuCypher protocol’s economic design and parametrization, particularly over the long-term and with respect to the sustainability of the network. From a governance perspective, it is also amongst the most malleable. If a quorum of stakers wish to set prices outside the range, then they have the right to lobby and propose a widening of the global fee range, its removal altogether, or some other design modification (e.g. narrowing the range). They may do so via the NuCypher DAO – the owner of all NuCypher smart contracts -  by submitting a proposal which is voted on by stakers weighted in proportion to their stake size. See <LINK> for guidance on the NuCypher DAO and official NuCypher governance processes.
```